### PR TITLE
Add more ways to get in contact for help

### PR DESCRIPTION
**Pull request by SimonBarendse**
We're super eager to help anyone wanting to start contributing. I've added a link to book a pairing session with one of our teammates and another mention of the Slack space where we're hanging out to help.

In addition, I've added a section on community & support, where I've added the Slack space, the newsletter and filing issues, so it's clear for folks visiting the README how they can get involved and stay up-to-date, if they so wish.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,20 @@ No more plaintext credentials in your home directory. Automatically authenticate
...
 For the contribution guidelines, see [CONTRIBUTING.md](CONTRIBUTING.md).
 
+Not sure where or how to get started? We're happy to help! You can:
+- Book a free [pairing session](https://calendly.com/d/grs-x2h-pmb/1password-shell-plugins-pairing-session) with one of our teammates
+- Join the Developer [Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) workspace, and ask us any questions there
+
+## 💚 Community & Support
+
+- Join the Developer [Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) workspace
+- Subscribe to the [Newsletter](https://1password.com/dev-subscribe/)
```
Comment (florisvdg) [inline README.md:48]: How about:
```suggestion
- Subscribe to the [Developer Newsletter](https://1password.com/dev-subscribe/)
```

File: README.md [hunk 2]
```diff
@@ -34,14 +34,20 @@ No more plaintext credentials in your home directory. Automatically authenticate
...
-
 For the contribution guidelines, see [CONTRIBUTING.md](CONTRIBUTING.md).
 
+Not sure where or how to get started? We're happy to help! You can:
+- Book a free [pairing session](https://calendly.com/d/grs-x2h-pmb/1password-shell-plugins-pairing-session) with one of our teammates
+- Join the Developer [Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) workspace, and ask us any questions there
+
+## 💚 Community & Support
+
+- Join the Developer [Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) workspace
```
Comment (florisvdg) [inline README.md:47]: Not so nice to have the same line repeated right underneath each other:

image

**Review by AndyTitu — APPROVED** [review]
Awesome initiative!

**Review by florisvdg — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -34,14 +34,21 @@ No more plaintext credentials in your home directory. Automatically authenticate
...
 For the contribution guidelines, see [CONTRIBUTING.md](CONTRIBUTING.md).
 
+Not sure where or how to get started? We're happy to help! You can:
+- Book a free [pairing session](https://calendly.com/d/grs-x2h-pmb/1password-shell-plugins-pairing-session) with one of our teammates
+- Join the Developer [Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) workspace, and ask us any questions there
+
+## 💚 Community & Support
+
+- File an [issue](https://github.com/1Password/shell-plugins/issues/new/choose) for bugs and feature requests
+- Join the Developer [Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA) workspace
```
Comment (florisvdg) [inline README.md:48]: Tiny non-blocking suggestion:
```suggestion
- Join the [Developer Slack workspace](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA), and ask us any questions there

## 💚 Community & Support

- File an [issue](https://github.com/1Password/shell-plugins/issues/new/choose) for bugs and feature requests
- Join the [Developer Slack workspace](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA)
```
Makes it feel more like an actual destination than just "Slack".

Comment (florisvdg) [reply README.md]: Committing this to unblock the PR.

**Review by florisvdg — APPROVED** [review]

**Review by hculea — APPROVED** [review]

**Pull request merged**